### ISSUE TITLE
bug: tv series searched for disappears from result screen

### ISSUE THREAD

**Issue by diamaunt** [theotherp/nzbhydra#364]
when searching for tv and selecting the series from the dropdown, after the search, the series is blank, requiring another entry for another search  (eg, another episode or season of same show).

it would be nice if the results page kept the series search info populated. so as to make further searches easier.

**Comment by theotherp** [comment]
Yeah that would be nice but due to the way the search is implemented I'm not sure I can achieve this behavior. It would certainly take a lot of work (even though it seems to simple). I'll keep this open as a reminder, who knows.
